feeds/wlan-ap/opensync/files/bin/flash-firmware

```diff
@@ -40,6 +40,8 @@ uci commit
 /sbin/sysupgrade $IMGFILE
 if [ "$?" != "0" ] ; then
   echo "$0: Sysupgrade failed."
+  rm /tmp/sysupgrade.meta
+  rm /tmp/upgrade.tar
```

Comment:
I guess this could also be upgrade.bin for some targets?

Reply:
Ah, good catch. I'll change this to upgrade.*